.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
```

Comment:
Oops sorry I forgot to mention I don't think we should include your editors settings in our repo.

Reply:
> EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs. The EditorConfig project consists of a file format for defining coding styles and a collection of text editor plugins that enable editors to read the file format and adhere to defined styles. EditorConfig files are easily readable and they work nicely with version control systems.

see: http://editorconfig.org

Reply:
Can we make this a separate thing? I don't want to mix this with JSON benchmark contributions. Mainly I want to research what this file is for but I don't want to hold your benchmarks back because I think they are great :)